lib/image.coffee

```diff
@@ -12,6 +12,10 @@ emit = ($item, item) ->
 
 bind = ($item, item) ->
   $item.dblclick -> editor.textEditor $item, item
-  $item.find('img').dblclick -> dialog.open item.text, this
+  url = item.url
+  if item.ipfs?
+    $.get "http://localhost:8080/ipfs/#{item.ipfs}", ->
```

Comment:
I initiate the get when the page is rendered. I don't expect a result until the user double-clicks to enlarge the image. If ipfs is slow one can try later, then it will be fast because images are stored in the local node.